README.md

```diff
@@ -38,7 +38,22 @@ go get github.com/prometheus-community/prom-label-proxy
 
 ## How does this project work?
 
-This application proxies the `/federate`, `/api/v1/query`, `/api/v1/query_range`, `/api/v1/series`, `/api/v1/labels`, `/api/v1/label/<name>/values`, `/api/v1/rules`, `/api/v1/alerts` Prometheus endpoints as well as `/api/v2/silences` Alertmanager endpoint and it ensures that a particular label is enforced in the particular request and response.
+This application proxies the following endpoints and it ensures that a particular label is enforced in the particular request and response:
+
+* `/federate` for GET method (Prometheus)
+* `/api/v1/query_exemplars` for GET and POST methods (Prometheus)
```

Comment:
Thanos already supports `query_exemplars` API, which is the same as how it supports the federated Rules API.

Reply:
Thanks for the heads-up! I was too lazy to check :)
I've updated that line in #79